packages/client/src/LocClient.ts

```diff
@@ -382,7 +382,7 @@ export class LocMultiClient {
     }
 
     async fetchAllForVerifiedThirdParty(legalOfficers: LegalOfficerClass[]): Promise<LocRequest[]> {
-        const entries = await this.nodeApi.query.logionLoc.locsByVerifiedIssuerMap.entries(this.currentAddress);
+        const entries = await this.nodeApi.query.logionLoc.locsByVerifiedIssuerMap.entries(this.currentAddress.address);
```

Comment:
This call will only work with Polkadot accounts. With an Ethereum address, it will crash. But maybe the check is done upfront?